### PR TITLE
feat: support deployment statement

### PR DESCRIPTION
**Pull request by aceforeverd**
* **Please check if the PR fulfills these requirements**

- The commit message follows our guidelines
- Tests for the changes have been added (for bug fixes / features)
- Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
support
 - #460
 - #443 
 - #447  

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: hybridse/src/planv2/ast_node_converter.cc [hunk 1]
```diff
@@ -592,6 +599,28 @@ base::Status ConvertStatement(const zetasql::ASTStatement* statement, node::Node
...
+            }
+            *output = node_manager->MakeLoadDataNode(file_name, table_path, options);
+            break;
+        }
+        case zetasql::AST_DEPLOY_STATEMENT: {
+            const auto ast_deploy_stmt = statement->GetAsOrNull<zetasql::ASTDeployStatement>();
+            CHECK_TRUE(ast_deploy_stmt != nullptr, common::kSqlAstError, "not an ASTDeployStatement");
+            node::SqlNode* deploy_stmt = nullptr;
+            CHECK_STATUS(ConvertStatement(ast_deploy_stmt->stmt(), node_manager, &deploy_stmt));
+            *output = node_manager->MakeDeployStmt(ast_deploy_stmt->name()->GetAsString(), deploy_stmt,
```
Comment (aceforeverd) [inline hybridse/src/planv2/ast_node_converter.cc:662]: if_not_exists

File: hybridse/include/node/node_enum.h [hunk 2]
```diff
@@ -266,6 +272,8 @@ enum PlanType {
     kProjectNode,
     kPlanTypeCreateSp,
     kPlanTypeCreateIndex,
+    kPlanTypeLoadData,
+    kPlanTypeDeploy,
     kUnknowPlan = 100,
```
Comment (aceforeverd) [inline hybridse/include/node/node_enum.h]: -1

File: hybridse/include/node/plan_node.h [hunk 3]
```diff
@@ -435,6 +436,43 @@ class CmdPlanNode : public LeafPlanNode {
     std::vector<std::string> args_;
 };
 
+class DeployPlanNode : public LeafPlanNode {
+ public:
+    explicit DeployPlanNode(const std::string& name, const SqlNode* stmt, bool if_not_exist)
+        : LeafPlanNode(kPlanTypeDeploy), name_(name), stmt_(stmt), if_not_exist_(if_not_exist) {}
+    ~DeployPlanNode() {}
+
+    const std::string& name() const { return name_; }
```
Comment (aceforeverd) [inline hybridse/include/node/plan_node.h:449]: create a `getStmtSql`

File: hybridse/include/node/plan_node.h [hunk 4]
```diff
@@ -435,6 +436,43 @@ class CmdPlanNode : public LeafPlanNode {
...
+class LoadDataPlanNode : public LeafPlanNode {
+ public:
+    explicit LoadDataPlanNode(const std::string& f, const std::vector<std::string>& tp,
+                          const std::shared_ptr<ImportOptions> op)
+        : LeafPlanNode(kPlanTypeLoadData), file_(f), table_path_(tp), options_(op) {}
+    ~LoadDataPlanNode() {}
+
+    const std::string& file() const { return file_; }
+    const std::vector<std::string>& table_path() const { return table_path_; }
+    const std::shared_ptr<ImportOptions>& options() const { return options_; }
```
Comment (aceforeverd) [inline hybridse/include/node/plan_node.h]: `ImportOptions` naming -> `OptionsMap`

File: hybridse/include/node/plan_node.h [hunk 5]
```diff
@@ -435,6 +436,43 @@ class CmdPlanNode : public LeafPlanNode {
...
+
+class LoadDataPlanNode : public LeafPlanNode {
+ public:
+    explicit LoadDataPlanNode(const std::string& f, const std::vector<std::string>& tp,
+                          const std::shared_ptr<ImportOptions> op)
+        : LeafPlanNode(kPlanTypeLoadData), file_(f), table_path_(tp), options_(op) {}
+    ~LoadDataPlanNode() {}
+
+    const std::string& file() const { return file_; }
+    const std::vector<std::string>& table_path() const { return table_path_; }
```
Comment (aceforeverd) [inline hybridse/include/node/plan_node.h]: 改成 db & table

File: cases/plan/cmd.yaml [hunk 6]
```diff
@@ -140,4 +140,104 @@ cases:
...
-
+  - id: show_deployments
+    desc: show deployments
+    sql: SHOW DEPLOYMENTS;
+    expect:
+      node_tree_str: |
+        +-node[CMD]
+          +-cmd_type: show deployments
+          +-args: []
+  - id: shwo_deployment
```
Comment (aceforeverd) [inline cases/plan/cmd.yaml]: show

**Review by jingchen2222 — APPROVED** [review]
LGTM

**Review by dl239 — APPROVED** [review]

**Pull request merged**